packages/server/src/test/java/software/uncharted/terarium/hmiserver/service/data/ProjectSearchServiceTests.java

```diff
@@ -72,9 +73,12 @@ public void testItCanSearchPrivateProjectAsReader() throws Exception {
 		project.setDescription("my description");
 
 		projectSearchService.indexProject(project);
+		projectSearchService.forceESRefresh();
 		projectSearchService.addProjectPermission(project.getId(), searcherId);
+		projectSearchService.forceESRefresh();
```

Comment:
You only need one at the end of the logic if you are doing forced-refresh index-wide.